jina/executors/crafters/__init__.py

```diff
@@ -9,10 +9,15 @@
 
 
 class BaseCrafter(BaseExecutor):
-    """A :class:`BaseCrafter` transforms the content of `DocumentProto` or `Chunk`. It can be used for preprocessing,
-    segmenting etc. It is an interface for Crafters which is a family of executors intended to apply
+    """
+    A :class:`BaseCrafter` transforms the content of `DocumentProto`.
```

Comment:
```suggestion
    A :class:`BaseCrafter` transforms the content of `Document`.
```

I think here we are dealing with `Document`s themselves, no?